utils/build_swift/driver_arguments.py

```diff
@@ -224,22 +224,39 @@ def _apply_default_arguments(args):
 def create_argument_parser():
     """Return a configured argument parser."""
 
+    # NOTE: USAGE, DESCRIPTION and EPILOG are defined at the bottom of the file
     parser = _ApplyDefaultsArgumentParser(
         apply_defaults=_apply_default_arguments,
         formatter_class=argparse.RawDescriptionHelpFormatter,
         usage=USAGE,
         description=DESCRIPTION,
         epilog=EPILOG)
 
+    builder = parser.to_builder()
+
+    # Prepare DSL functions
+    option = builder.add_option
```

Comment:
Why change this to "option" instead of using "add_option"?

Reply:
Calling it just `option` is shorter which cuts down on visual noise and it's more declarative inline with the rest of the DSL.